drivers/atwinc15x0/Kconfig

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_ATWINC15X0
+    bool
+    prompt "ATWINC15x0 WiFi module" if !(MODULE_NETDEV_DEFAULT && HAVE_ATWINC15X0)
+    default y if (MODULE_NETDEV_DEFAULT && HAVE_ATWINC15X0)
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_GPIO_IRQ
+    depends on HAS_PERIPH_SPI
+    depends on TEST_KCONFIG
+    select PACKAGE_DRIVER_ATWINC15X0
+    select MODULE_PERIPH_GPIO
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_SPI
+    select MODULE_NETDEV_ETH
+    select MODULE_ZTIMER_MSEC
+    help
+        This netdev device driver is used for the Microchip ATWINC15x0
+        WiFi module. The ATWINC15x0 WiFi module is widely used as WiFi
+        interface on various boards and extension boards.
+
+if MODULE_ATWINC15X0
+
+config WIFI_SSID
+    string "WiFi SSID"
+    default "RIOT_AP"
+    help
+        SSID of the AP to be used. The SSID must not contain more
+        than 32 characters.
+
+config WIFI_USE_WPA2
+        bool "Use WPA2 authentication"
+        default n
+        help
+            Specify whether WPA2 authentication is used to connect to the
+            AP. If WPA2 authentication is enabled, the passphrase has to be
+            defined.
+
+config WIFI_PASS
+        string "Passphrase for WPA2 Personal Mode authentication"
+        depends on WIFI_USE_WPA2
+        default "This is RIOT-OS"
+        help
+            The passphrase is defined as a clear text string with a maximum
+            of 64 characters. It is used for WPA2 personal mode authentication.
```

Comment:
Definitely outside the scope of this PR, but at some point we need to handle this system-wide and not driver-specific.

Reply:
Hm, this seems to be a hen's egg problem because it is required for the compilation of the driver.

Reply:
Yeah, as it is currently done. Perhaps we can think about some key management (something similar to [credman](https://doc.riot-os.org/group__net__credman.html)?). Would it be possible (on a separate PR) to start unifying the macro name and have one system-wide for other modules (e.g., ESP)?